release_notes/368-release-notes.adoc

```diff
@@ -0,0 +1,137 @@
+=== Vulnerability triage workflows
+{product-title} 3.68 includes the ability to triage vulnerabilities in a variety of ways to support your vulnerability management process.
+See xref:../operating/manage-vulnerabilities.adoc#vulnerability-management-proccess_acs-operating-manage-vulnerabilities[Managing vulnerabilities] for more information.
+
+[id="report-scheduling"]
+=== Report scheduling for vulnerabilities
+{product-title} 3.68 includes the ability to schedule reports for vulnerabilities helping you to send scheduled communications to key stakeholders to assist in the vulnerability management process. See xref:../operating/manage-vulnerabilities.adoc#vulnerability-management-reporting_acs-operating-manage-vulnerabilities[Reporting vulnerabilities to teams] for more information.
+
+[id="use-aws-ecr-assumeroles"]
+=== Use AWS ECR AssumeRoles
+AWS AssumeRoles allows you to define roles with specific permissions and then granting users access to those roles. {product-title} 3.68 includes the ability to use AWS ECR AssumeRoles to configure roles and grant various levels of access to users.
+For more details, see xref:../integration/integrate-with-image-registries.adoc#use-assumerole-with-ecr[Using assumerole with Amazon ECR].
+
+[id="output-format-enhancements"]
+=== Enhancements for CI outputs
+Red Hat has improved the usability of {product-title} CI integrations. CI outputs now show additional detailed information about the vulnerabilities and the security policies responsible for broken builds.
+
+[id="inactive-software-components"]
+=== Inactive Software Component Identification
+Red Hat has improved the ability of users to identify if a software package that is being used inside of a container image is inactive, which helps users to triage if a software package may be able to be removed as a hardening step or vulnerability remediation.
+
+[id="known-issue"]
+== Known issue
+When the same software package is installed in multiple layers in a docker image {product-title-short} may incorrectly identify the state of that package.
+
+[id="important-bug-fixes"]
+== Important bug fixes
+
+* *ROX-8709*: Previously, searching for CVE's with a specific severity did not returned any results. This issue has been fixed.
+* *ROX-8983*: Previously, when configuring the *Manage Watches* feature, if you added more than 12 images to the watch list, the image list would not display properly. This issue has been fixed.
+* *ROX-8276*: Previously, when the {product-title-short} Operator accessed the `central-htpasswd` secret, it would create a false positive policy violation for the *OpenShift: Advanced Cluster Security Central Admin Secret Accessed* default policy. This issue has been fixed.
+
+[id="security-update"]
+== Security update
+In earlier versions of {product-title}, the `write` permission for the `APIToken` resource allowed users to create API tokens for any role, including the `admin` role. This issue has been fixed.
```

Comment:
Is it just the admin role you're removing the ability to change? If so, consider:
```suggestion
In earlier versions of {product-title}, the `write` permission for the `APIToken` resource allowed users to create API tokens for any role, including the `admin` role. This issue has been fixed, and it is no longer possible to create API tokens for the `admin` role..
```